### ISSUE TITLE
The screenshot viewer should be opened using keyboard navigation

### ISSUE THREAD

**Issue by ioanarusiczki** [mozilla/addons-frontend#9368]
STR:
1. Go to an add-on detail page which contains images 
https://addons.allizom.org/en-US/firefox/addon/noscript/?src=homepage-collection-featured
2. Navigate using Tab key from the ratings section to the screenshot viewer.
3. Try to open the first screenshot using Enter/Space

Expected result:
The image is available on the screen. 

Actual result:
The screenshot cannot be accessed. 

Notes:
Would be nice to support it.
Issue is reproducible on all AMO servers - FF75(Win10). 

![cannot open image](https://user-images.githubusercontent.com/33448286/79959785-36087300-848d-11ea-8f09-69791ad34ec9.gif)

**Comment by willdurand** [comment]
We use a third-party library for the photo gallery. Hopefully we'll be able to open it with keyboard navigation.
